itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -376,4 +386,63 @@ public void testIncrementalMetadataReplication() throws Throwable {
             "custom.value\t "
         });
   }
+
+  @Test
+  public void testBootStrapDumpOfWarehouse() throws Throwable {
+    String randomOne = RandomStringUtils.random(10, true, false);
+    String randomTwo = RandomStringUtils.random(10, true, false);
+    String dbOne = primaryDbName + randomOne;
+    String dbTwo = primaryDbName + randomTwo;
+    WarehouseInstance.Tuple tuple = primary
+        .run("use " + primaryDbName)
+        .run("create table t1 (i int, j int)")
+        .run("create database " + dbOne)
+        .run("use " + dbOne)
+        .run("create table t1 (i int, j int) partitioned by (load_date date) "
+            + "clustered by(i) into 2 buckets stored as orc tblproperties ('transactional'='true') ")
+        .run("create database " + dbTwo)
+        .run("use " + dbTwo)
+        .run("create table t1 (i int, j int)")
+        .dump("`*`", null, Arrays.asList("'hive.repl.dump.metadata.only'='true'",
```

Comment:
Do we support incremental dump with * for dbName? From the code, I see it will not work as events filter directly matches to dbName input which doesn't match to any of the events.

Reply:
havent checked this i thought i thought it would work will write a test case for this as well.

Reply:
cant write test for this since we dont have ability to have two different rdbms backing  in metastore.